base/Dockerfile

```diff
@@ -56,4 +66,12 @@ RUN curl -LO "https://dl.k8s.io/release/v${KUBE_VERSION}/bin/linux/amd64/kubectl
 RUN ssh-keyscan github.com 2>/dev/null > /etc/ssh/ssh_known_hosts
 
 USER coder
+
+# Install pyenv that has been requested by the clinical team.
+ENV PYENV_ROOT="/home/coder/.pyenv"
+ENV PATH="${PYENV_ROOT}/shims:${PYENV_ROOT}/bin:${PATH}"
+# Latest release as of 2022-12-12 (https://github.com/pyenv/pyenv/releases)
+ENV PYENV_GIT_TAG=v2.3.8
```

Comment:
I’d make this an ARG not ENV

potentially true for PYENV_ROOT also. Is it a variable required only at install time or when running the container too?

Reply:
Also just be mindful of notifying staff depending what we end up doing here. There's a good chance people have done there own pyenv thing. E.g. I have.

Reply:
@dcarrion87 Could you let me know what is different/custom in your setup ? @anaoum Good point. I think PYENV_GIT_TAG could be made as an ARG. Shall double check on PYENV_ROOT to see if it isn't needed at run-time.

Reply:
@anaoum & @andychisholm could you let me know if your preference would also be to leave out pyenv and just install its dependencies as recommended by @dcarrion87 ?

Reply:
@arunkumarpandian I agree -- this feels like an analogous situation to direnv.

Let's "support" it with all the packages it needs but not actually install/configure it for users; given it's happy install path seems to be at odds with our persistence pattern.

Reply:
OK. shall just add the dependencies into the image.